impexp-config/src/main/java/org/citydb/config/project/kmlExporter/KmlExporter.java

```diff
@@ -247,7 +251,10 @@ public KmlExporter() {
 		callbackNameJSONP = "handle_3DCityDB_data";
 		createGltfModel = false;
 		notCreateColladaFiles = false;
-		embedTexturesInGltfFiles = false;
+		embedTexturesInGltfFiles = true;
+		exportGltfBinary = false;
+		enableGltfDracoCompression = true;
+		exportGltfV1 = false;
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).